demos/sample_azure_iot_pnp/sample_azure_iot_pnp_core.c

```diff
@@ -0,0 +1,618 @@
+/* Copyright (c) Microsoft Corporation. All rights reserved. */

```

Comment:
This is a split from the original sample_azure_iot_pnp.c, containing only the code that deals with the middleware functions **and the functions defined in the interface header sample_azure_iot_pnp.h**